### ISSUE TITLE
Trezor not recognized during setup

### ISSUE THREAD

**Issue by alek47TU** [Electron-Cash/Electron-Cash#1787]
Hi

My Trezor T is working fine with Wallet through Chrome and Firefox, it is not recognized by the Electron setup process.

Please help!

Thanks all

**Comment by cculianu** [comment]
What platform/OS?

**Comment by alek47TU** [comment]
Hi Calin

Thanks, MacOS.


Best


Shane

**Comment by cculianu** [comment]
I'm on macOS too.  What exact version of the OS (Mojave, Catalina, High Sierra, etc) and what version of Electron Cash?

**Comment by alek47TU** [comment]
10.15.2 and the latest Electron, I just downloaded it today. I’m trying to sweep my forked coins, might be easier through Binance.


Best


Shane

**Comment by cculianu** [comment]
I actually am avoiding Catalina.  I run Mojave.  Let me install Catalina on another laptop here and see if I can reproduce this problem.

**Comment by alek47TU** [comment]
Thanks, I have Mojave machines I’ll also try today. Otherwise I’m stuck! Binance and Coinex have both delisted BSV, seems that Electron is my only option now?


Best


Shane

**Comment by cculianu** [comment]
Oh, wait.  You want to use Electron Cash to redeem BSV?  You can't do that with modern versions of Electron Cash anymore.  The last version that could do that I believe was 3.3.4CS. 

You instead should go here and get the ElectrumSV wallet:  https://electrumsv.io

If you do have time though, can you please still try this on Mojave and let me know if it works better?  I'm curious if the issue is Catalina or something else.  That would be very helpful.

Thank you.

**Comment by EchterAgo** [comment]
You are wrong about CoinEx having delisted BSV, they still list it and still auto split any incoming funds.

**Comment by alek47TU** [comment]
Tech support this morning stated it is delisted, my BSV receive link is grey. Please advise how to do it! Thanks 😊 


Best


Shane

**Comment by alek47TU** [comment]
ElectrumSV loads on Catalina but the addresses it generates are “invalid” to use, Trezor rejects them. The faucet they suggest using says it’s empty!

I’ll try with Mojave later today and circle back.

Anyone with any ides how to claim my BSV would be gratefully received!


Cheers

Shane

**Comment by EchterAgo** [comment]
I get a deposit address just fine. The BSV markets are still open and actively traded. There is no delisting news like CoinEx usually does when delisting coins. Last time I deposited BSV was 7 days ago. You do mean https://www.coinex.com/ right?

> Tech support this morning stated it is delisted, my BSV receive link is grey. Please advise how to do it! Thanks 😊 Best Shane
> […](#)
> On Jan 28, 2020, at 06:51, Axel Gembe ***@***.***> wrote: ﻿ You are wrong about CoinEx having delisted BSV, they still list it and still auto split any incoming funds. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by EchterAgo** [comment]
Their faucet is crap. Either it times out or is empty.

If you want I can send you some BCH dust so you can split your BCH coins from your BSV coins first. After that you'd have to raise an issue in the Electrum SV repository regarding their Trezor issues.

**Comment by alek47TU** [comment]
That makes no sense! See screenshots from today..

**Comment by alek47TU** [comment]
Thank you, I have BTC and BCH. Without a BSV receive address I’m confused how to send anything and solve this. Brain is hurting, am I missing something simple??!

**Comment by EchterAgo** [comment]
I can't see your screenshots, probably github doesnt like image attachments. 

You don't need a BSV address to split your coins. It would go like this:

* I send you some BCH dust that is already split.
* You use the Trezor app to send your whole (this is important) BCH balance to yourself, to a new receive address.
* The funds are now split, you can move them independently.

This doesn't solve that you need some way to move your BSV though.

 You might want to come to the EC telegram channel, lots of people there can help: https://t.me/electroncashwallet
